src/Views/Form/Templates/Classic/resources/views/header.php

```diff
@@ -2,42 +2,62 @@
         <p class="give-form-description"><?= $description ?></p>
         <?php if ($isSecureBadgeEnabled) : ?>
-        <aside class="give-form-secure-badge">
-            <svg class="give-form-secure-icon">
-                <use href="#give-icon-lock" />
-            </svg>
-            <?= $secureBadgeContent ?>
-        </aside>
+            <aside class="give-form-secure-badge">
+                <svg class="give-form-secure-icon">
+                    <use href="#give-icon-lock"/>
+                </svg>
+                <?= $secureBadgeContent ?>
+            </aside>
         <?php endif; ?>
     </div>
-    <aside class="give-form-stats-panel">
-        <ul class="give-form-stats-panel-list">
-            <li class="give-form-stats-panel-stat">
-                <span class="give-form-stats-panel-stat-number">$0</span> raised
-            </li>
-            <li class="give-form-stats-panel-stat">
-                <span class="give-form-stats-panel-stat-number">50</span> donations
-            </li>
-            <li class="give-form-stats-panel-stat">
-                <span class="give-form-stats-panel-stat-number">$10,000</span> goal
-            </li>
-            <li class="give-form-goal-progress">
-                <div class="give-form-goal-progress-meter" role="meter" aria-labelledby="meter-label" aria-valuemin="0" aria-valuemax="1000000" aria-valuenow="200000" aria-valuetext="20%">
-                    <div id="meter-label" hidden>
-                        $20,000 of $1,000,000 goal
+    <?php if ($hasGoal) : ?>
+        <aside class="give-form-stats-panel">
+            <ul class="give-form-stats-panel-list">
+                <li class="give-form-stats-panel-stat">
+                    <span class="give-form-stats-panel-stat-number">
+                        <?= $goalStats[ 'raised' ]; ?>
+                    </span> <?= __('raised', 'give'); ?>
+                </li>
+                <li class="give-form-stats-panel-stat">
+                    <span class="give-form-stats-panel-stat-number">
+                         <?= $goalStats[ 'count' ]; ?>
+                    </span> <?= $goalStats[ 'countLabel' ]; ?>
+                </li>
+                <li class="give-form-stats-panel-stat">
+                    <span class="give-form-stats-panel-stat-number">
+                        <?= $goalStats[ 'goal' ]; ?>
+                    </span> <?= __('goal', 'give'); ?>
+                </li>
+                <li class="give-form-goal-progress">
+                    <div
+                        role="meter"
+                        class="give-form-goal-progress-meter"
+                        style="--progress: <?= $goalStats[ 'progress' ]; ?>%"
+                        aria-labelledby="meter-label"
+                        aria-valuemin="0"
+                        aria-valuemax="<?= $goalStats[ 'goal' ]; ?>"
+                        aria-valuenow="<?= $goalStats[ 'raised' ]; ?>"
```

Comment:
I realized after my first review that these two values should actually both be unformatted values. I think we will need to add new entries to the `$goalStats` array for both of these.

Reply:
Resolved e8876edd